### PR TITLE
[navigation] Add workspace icon to left nav / workspace picker menu / home page.

### PR DESCRIPTION
**Pull request by SuZhou-Joe**
### Description

We have icons for each user case and they should be displayed in home page / workspace picker menu and left nav. This PR is to adopt the icons in related places.

### Use case icons in home page
image

### Use case icons in workspace picker
image

### Use case icons in left nav
image

## Changelog

- feat: Add workspace icon to left nav / workspace picker menu / home page.

### Check List

- [ ] All tests pass
  - [ ] `yarn test:jest`
  - [ ] `yarn test:jest_integration`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [ ] Update [CHANGELOG.md](./../CHANGELOG.md)
- [ ] Commits are signed per the DCO using --signoff

### CODE REVIEW

File: src/core/public/chrome/ui/header/collapsible_nav_group_enabled_top.tsx [hunk 1]
```diff
@@ -125,12 +152,10 @@ export const CollapsibleNavTop = ({
         <>
           <EuiSpacer />
           <EuiText>
-            <div className="nav-link-item" style={{ fontWeight: 'normal' }}>
-              {currentNavGroup?.title}
-            </div>
+            <div style={{ fontWeight: 'normal' }}>{currentNavGroup?.title}</div>
```
Comment (ruanyl) [inline src/core/public/chrome/ui/header/collapsible_nav_group_enabled_top.tsx:155]: Q: what's this title? why customize the font-weight?

File: src/plugins/workspace/public/components/use_case_overview/setup_overview.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+.analyticsGettingStartedWorkspaceCardsIcon {
+  color: $euiColorMediumShade;
```
Comment (ruanyl) [inline src/plugins/workspace/public/components/use_case_overview/setup_overview.scss:2]: Should it just use the `color` props of `<OuiIcon />`?

**Review by ruanyl — APPROVED** [review]

File: src/plugins/workspace/public/components/use_case_overview/setup_overview.scss (hunk 2 above)
Comment (SuZhou-Joe) [reply src/plugins/workspace/public/components/use_case_overview/setup_overview.scss:2]: image
Actually I tried all of the possible values here but none of the color meet our expectation so I had to create scss file for each icon.

**Review by ruanyl — DISMISSED** [review]

File: src/core/public/chrome/ui/header/collapsible_nav_group_enabled_top.tsx (hunk 1 above)
Comment (SuZhou-Joe) [reply src/core/public/chrome/ui/header/collapsible_nav_group_enabled_top.tsx]: Nice catch, updated and thanks for that.

**Review by ZilongX — APPROVED** [review]

**Pull request merged**